tests/functional/z_cluster/cluster_expansion/test_create_multiple_device_classes.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    ignore_leftovers,
+    tier1,
+    brown_squad,
+    skipif_no_lso,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_osd_running_nodes
+from ocs_ci.helpers.multiple_device_classes import (
+    create_new_lvs_for_new_deviceclass,
+    verification_steps_after_adding_new_deviceclass,
+)
+from ocs_ci.ocs.resources.pv import wait_for_pvs_in_lvs_to_reach_status
+from ocs_ci.ocs.resources.storage_cluster import (
+    add_new_deviceset_in_storagecluster,
+    get_storage_cluster,
+)
+from ocs_ci.helpers.helpers import (
+    create_ceph_block_pool_for_deviceclass,
+    create_deviceclass_storageclass,
+)
+from ocs_ci.ocs.cluster import check_ceph_pool_exist
+from ocs_ci.utility.utils import ceph_health_check, TimeoutSampler
+
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier1
```

Comment:
I don't think these tests are considered function for the product. I would say these are more advanced scenarios and we can mark them with tier2

Reply:
Why not? This is something we expect from the client, like adding capacity or resizing the osd.

Reply:
ok, we can keep as is

Reply:
Ack